code/controllers/subsystem/SSticker.dm

```diff
@@ -737,11 +737,14 @@ SUBSYSTEM_DEF(ticker)
 	if(end_string)
 		end_state = end_string
 
-	// Play a haha funny noise
+	// Play a haha funny noise for those who want to hear it :)
 	var/round_end_sound = pick(GLOB.round_end_sounds)
 	var/sound_length = GLOB.round_end_sounds[round_end_sound]
-	SEND_SOUND(world, sound(round_end_sound))
-	sleep(sound_length)
+
+	for(var/mob/M in GLOB.player_list)
+		if(M.client.prefs.sound & SOUND_END_OF_ROUND)
+			SEND_SOUND(M, round_end_sound)
+			sleep(sound_length)
```

Comment:
Wasn't sure if it would sleep it for them or for everyone. I had the same thought and you might be right, I only had myself to test this with.
I actually saw in CONTRIBUTING.md that ```addtimer()``` is preferred to ```sleep()```, so was considering making that swap too.

Reply:
Yeah, you shouldn't add a timer or sleep here. This will indeed sleep for every player.

Reply:
fixed!